### ISSUE TITLE
Requires fastboot?

### ISSUE THREAD

**Issue by scottkidder** [mirego/ember-best-language#4]
```
ember.debug.js:26509 Error: Assertion Failed: Attempting to inject an unknown injection: 'service:fastboot'
    at new EmberError (ember.debug.js:13653)
    at assert (ember.debug.js:13895)
    at Registry.validateInjections (ember.debug.js:10792)
    at FactoryManager.create (ember.debug.js:10399)
    at Proxy.create (ember.debug.js:10168)
    at instantiateFactory (ember.debug.js:10256)
    at lookup (ember.debug.js:10213)
    at Container.lookup (ember.debug.js:10075)
    at Class.lookup (ember.debug.js:37804)
    at Class.injectedPropertyGet (ember.debug.js:29596)
```

**Comment by charlesdemers** [comment]
Hi! Yes as of now the addon requires FastBoot. I haven’t tested it in non-FastBoot apps but I plan to in the next days/weeks.

**Comment by basz** [comment]
I can confirm that even while fastboot is required, running `FASTBOOT_DISABLED=true ember serve` works.

I think it is possible to avoid using [`fastboot: Ember.inject.service(),`](https://github.com/mirego/ember-best-language/blob/1fc86b323b8fb4cacb621c52bb8ec0780ee1418a/addon/services/best-language.js#L6) so this hard dependency isn't necessary. 

It's explained here: https://ember-fastboot.com/docs/addon-author-guide#accessing-the-fastboot-service

Might help with adoption for non fastboot users

**Comment by charlesdemers** [comment]
This is fixed in #6, you can install version 0.1.0 to use it without FastBoot
